taxi/forms.py

```diff
@@ -50,3 +50,33 @@ def validate_license_number(
         raise ValidationError("Last 5 characters should be digits")
 
     return license_number
+
+
+class ManufacturerSearchForm(forms.Form):
+    name = forms.CharField(
+        max_length=255,
+        required=False,
+        label="",
+        widget=forms.TextInput(attrs={"placeholder": "Search by name "})
+    )
```

Comment:
I think this formatting can be improved. You can also use [black](https://black.readthedocs.io/en/stable/) to handle formatting for you.